### PR TITLE
Setup spec harness

### PR DESCRIPTION
**Pull request by jpasko1112**
Organized some data and added a runner file.

### CODE REVIEW

**Comment by garussell** [comment]
Looks like we are ready to go.

**Review by danieljsauer — APPROVED** [review]
Yep, this should work!

File: runner.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+
+locations = {
+  games: game_path,
+  teams: team_path,
+  game_teams: game_teams_path
+}
+
+stat_tracker = StatTracker.from_csv(locations)
+
+require 'pry'; binding.pry
```
Comment (garussell) [inline runner.rb:15]: Do we need the pry there?

**Review by garussell — APPROVED** [review]
Looks good.

**Comment by TLamb32** [comment]
Lets get started.

**Pull request merged**